src/Uno.Extensions.Maui.UI/Extensibility/IMauiEmbeddingExtension.cs

```
@@ -42,18 +42,18 @@ public interface IMauiEmbeddingExtension
 	/// <param name="app">The application instance.</param>  
 	/// <param name="window">The platform-specific window instance.</param>  
 	/// <returns>The built Maui application.</returns>  
-	MauiApp BuildMauiApp(MauiAppBuilder builder, Application app, Microsoft.UI.Xaml.Window window) => throw new PlatformNotSupportedException();
+	MauiApp BuildMauiApp(MauiAppBuilder builder, Application app, Microsoft.UI.Xaml.Window window) => default!;
```

`default!` can silently fail here here no?